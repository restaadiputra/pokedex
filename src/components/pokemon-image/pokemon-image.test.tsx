import React from 'react';
import { render } from '@testing-library/react';
import PokemonImage from './pokemon-image';

test('should render without error', () => {
  const { container } = render(
    <PokemonImage src="https://via.placeholder.com/150" alt="test" />
  );
  expect(container).toMatchSnapshot();
});
