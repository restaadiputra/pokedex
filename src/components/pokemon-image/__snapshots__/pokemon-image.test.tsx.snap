// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without error 1`] = `
<div>
  <div
    class="p-12 pt-8"
  >
    <div
      class="relative"
      style="padding-bottom: 100%;"
    >
      <img
        alt="test"
        class="absolute w-full h-full object-cover"
        src="https://via.placeholder.com/150"
      />
    </div>
  </div>
</div>
`;
